### ISSUE TITLE
 Scalac can't generate 55(jvm11) bytecode version

### ISSUE THREAD

**Issue by chucheng92** [scala/bug#12588]
## Reproduction steps

I try to use scalac compile (environment bounding jdk is jdk11) scala source code, but the generate bytecode version always be 52(jdk8). My scala version is 2.12.15.  I notice that scalac help show target: 

```
-target:<target> Target platform for object files. All JVM 1.5 - 1.7 targets are deprecated. (jvm-1.5,jvm-1.6,jvm-1.7,[jvm-1.8])
```

so, my question is scala-2.12 now whether didnot support generate jvm-11(55) or higher bytecode ?  whether user must use 2.13 or higher version?

## Problem

didnot support generate jvm-11 (55) or higher bytecode.

**Comment by som-snytt** [comment]
I don't know, I thought the getting started page had guidance on platform support, but now I can't find it again, so I'm just frustrated again with the docs.

The output of `scalac -help` is definitive.

There is a newer PR that says to prefer `-release` https://github.com/scala/scala/pull/9982 which they will hopefully merge in 2.13.9. That says that if you're targeting platform N you probably also intend to target its API.

**Comment by SethTisue** [comment]
https://github.com/scala/scala/pull/9916 and https://github.com/scala/scala/pull/10000 will be released in Scala 2.12.16, which we may publish as soon as this week.

**Comment by SethTisue** [comment]
If you want to test this support now, you could try a nightly build: https://stackoverflow.com/questions/40622878/how-do-i-tell-sbt-or-scala-cli-to-use-a-nightly-build-of-scala-2-12-or-2-13

**Comment by chucheng92** [comment]
@SethTisue OK, Thanks.

**Comment by chucheng92** [comment]
@SethTisue Hi, the background is that apache flink community discuss upgrade their scala version to 2.12.x. I suggested 2.12.16 because 2.12.16 backport jvm11,jvm17 target jvm functionality which the flink is needed.  So i wonder what's the timeline of 2.12.16 released version? I have just noticed a v2.12.16 tag.

**Comment by SethTisue** [comment]
You can track progress at https://contributors.scala-lang.org/t/scala-2-12-16-release-planning/5692
